docs/book/features/volume_health.md

```diff
@@ -0,0 +1,112 @@
+# vSphere CSI Driver - Volume Health
+Higher level advanced service such as Persistent Service operator wants to know the health status of persistent volumes so that it can make intelligent decisions.  Volume Health feature provides the health status of persistent volume to the higher level advanced service. Currently, Volume Health is only supported for Project Pacific Cluster and Tanzu Kubernetes Grid Cluster and only supported for block volume.
```

Comment:
Since we don't have a section explaining what is "Persistent Service", it may be confusing to mention it here.  I think it is better to remove the first sentence and remove "to the higher level advanced service" from the 2nd sentence.

Reply:
+1 to Xing's comment. Why not just talk about what is volume health here rather than explaining who is using it?